### ISSUE TITLE
Policy for accepting pull requests

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-code-registry#8]
Besides passing automatic checks that we set up, for new pull requests, it was suggested that we require

 * computers: the output of `verdi computer test <computername>`
 * codes: output of `ls -l /absolute/path/to/executable`

**Comment by ltalirz** [comment]
Another question is: do we accept only codes compiled by the HPC centre staff or do we also include user-compiled codes (which might be deleted without notice)?

I would say it is ok to accept PRs for user-compiled codes as long as there is some maintenance promise behind, i.e. an intention to inform users of further developments (including an update of this repo).

**Comment by yakutovicha** [comment]
Just copy-pasting the @sphuber 's question in the PR #22 : 

> Just a high-level question on the scope of this repository that this PR triggered for me: this addition seems to me to be very specific as it targets an explicit allocation on a single machine, so only people with access to that allocation can use these configurations. I am not sure how many people this would service, but do we have any "limits" on what gets accepted here, or can anyone add any configuration? Could I add a configuration that is just useful for me alone, just to help me simplify configuring my AiiDA install on multiple machines? I am not sure if this has already been discussed and if there is a consensus @ltalirz

**Comment by ltalirz** [comment]
Since the purpose of this registry is to share computer configurations, I think the minimum requirement for accepting new entries should be that they
 * can be used by other AiiDA users
 * can be expected to continue working for some time going forward (for how much we can define; e.g. I'd say at least one year)

This in my view excludes codes compiled by specific users unless there is a promise that they will be maintained,
and it excludes temporary scheduler partitions that are going to be deleted.

However, I believe there will always be some gray area, and we cannot expect that all users will be able to use all configurations here out of the box.

The good thing is that modifying configurations for your own purposes is very easy - you simply specify the ``--config ...`` option *together*  with the options for values you would like to overwrite.

**Comment by sphuber** [comment]
So that would place PR #22 in the gray zone if I understand correctly. I am fine with a gray zone because I don't think there is going to be a way around it, but we should make these guidelines clear in the `README.md` maybe. Also @yakutovicha the configuration you are adding, who will be using this?

**Comment by ltalirz** [comment]
I'll create a corresponding PR template then

**Comment by yakutovicha** [comment]
> Also @yakutovicha the configuration you added, who will be using this?

Yes, that is going to be used for the LSMO course. They will use AiiDAlab to perform the isotherm simulations.
